README.md

```diff
@@ -5,53 +5,62 @@ protopipe
 
 A pipeline prototype for the Cherenkov Telescope Array.
 
-Installation:
--------------
+Installation
+------------
 
 Get the source code and create the required basic conda environment:
 
           git clone https://github.com/cta-observatory/protopipe.git
           cd protopipe
           conda env create -f protopipe_environment.yml
           conda activate protopipe
+          
+In case you have already an environment 'myenv' set up for that kind of development, use:
 
+          conda env update -n myenv -f protopipe_environment.yml
+          
 This environment contains the bare minimum in order to run the scripts and build the documentation.
 
 It doesn't take into account any additional tool you could use later on (it is suggested to install _ipython_, _jupyter_ and _vitables_, especially if you want to contribute to the code).
 
-Next you need to install _protopipe_ itself:
+Next you need to install _protopipe_ itself (_protopipe_ is the folder where the code was expanded. The name differs if you downloaded a released tagged version, it can be _protopipe-0.2_ for instance) :
 
           cd protopipe
           python setup.py develop
 
 This will let you make changes to your local git repository without the need to update your environment every time.
+In the abscence of a _conda_ or _pip_ installation made avalaible, to install a released version (no further development), use :
+
+          python setup.py install
 
 Remember that the environment needs to be activated in order for _protopipe_ to work.
 This procedure has been successfully tested on macOS (10.10.5 & 10.14.6) and on Scientific Linux 7.
 
-Building the documentation:
----------------------------
+Building the documentation
+--------------------------
 
-Starting from your _protopipe_ local repository,
+From the main folder go down to the documentation repository and create the documentation :
 
-          cd protopipe
           cd docs
           make html
 
 You will probably get some harmless warnings.
+The initial page is stored in _ _build/html/index.html_, which you can open using your favorite internet browser.
 
-The initial page is stored in _build/html/index.html_, which you can open using your favorite internet browser.
+Test if it works
+-----------------
+Before starting to use protopipe, be sure to be inside the relevant environment (e.g. """conda activate protopipe""").
+Get a _simtel_ Monte Carlo file (mydata.gz) obtained with _prod3b_ anth the La Palma site and produce a simple DL1 output (It will not work for a paranal file):
 
-Basic use:
-----------
+          cd protopipe/scripts
+          python write_dl1.py -h
+          
+To be completed (it crashes so far)...
```

Comment:
In which conditions does this happen (OS, protopipe version, etc..)?
Perhaps it is best to not report bugs in the README and open a separate issue instead

Reply:
README has been updated to refer to issue #23